applications/openshift/scheduler/scheduler_address_argument/rule.yml

```diff
@@ -18,13 +20,12 @@ rationale: |-
 
 severity: medium
 
-ocil_clause: '<tt>--address</tt> argument is not set to the IP address of the loopback device'
+ocil_clause: 'scheduler network address is not set to the IP address of the loopback device'
 
 ocil: |-
     Run the following command on the master node(s):
-     <pre>$ ps -ef | grep kube-apiserver</pre>
-
-    Verify that the <tt>--address</tt> argument is set to <tt>127.0.0.1</tt>.
+    <pre>$ sudo grep -A2 address /etc/origin/master/master-config.yaml</pre>
+    The output should return <tt>false</tt>.
```

Comment:
The default is loopback/127.0.0.1, but some admins may manually specify it anyway. Don't want that to be a false finding.

Could change the OCIL clause to "If there is output, the IP address of the loopback device should be returned. No output indicates OpenShift is configured to the default setting of 127.0.0.1, which satisfies this requirement"... probably can word better than I.